docs/source/explanations/roadmap.md

```diff
@@ -0,0 +1,26 @@
+# Roadmap
+
+## Path to v0.1.0 Release
+
+We aim to release v0.1.0 in August 2025.
+
+- Support for reading from and writing to blob storage
+- Adoption of "closure table" pattern for expressing
+  tree structure of nodes with better performance and
+  referential integrity
+- Experimental support for streaming live data over
+  websockets
+- A read-only view of array data compatible with Zarr v2,
+  suct that Zarr readers can view Tiled as a Zarr store.
```

Comment:
I think all of these are 0.2.0 or beyond, but to me 0.1.0 was a while ago and was the initial serving of directories.

I've been pushing DLS towards rigid use of https://github.com/ytanikin/PRConventionalCommits to assist with following https://semver.org/ - reading/writing to blob storage is a `feat` (0.2.0), adjusting expression of trees is a `perf` or `refactor` (0.2.1), support for websockets is a `feat` (0.3.0), Zarr v2 reader is a `feat` (0.4.0).

I'm being rigid with semVer to make it easier for support teams to adopt changes in blueapi and know when they're potentially disruptive- currently we're happy to release potentially multiple times per day, which I recognise is quite a change in behaviour. But I was asked for thoughts ;)

Reply:
Remember, you're not bounded by 0.99.0 if you think you'll still be making breaking changes after that, while I'm also going to then push for 1.0.0 at some point, 0.1.0 actual shouldn't be scary!